content/docs/04-clusters/03-edge.md

```diff
@@ -51,6 +51,14 @@ Palette Edge Platform provides two solutions for edge, designed to support large
 
 Designed for sites that typically have a single large bare-metal appliance. Virtualized nodes are instantiated on the appliance using libvirt, and the desired version of OS and Kubernetes is deployed on the nodes. Each Virtual Machine (VM) represents a Kubernetes node. Users can specify placement settings for these virtual machines to ensure they are launched in the desired network and storage pools. Users can also configure VM hardware settings such as CPU, Memory, Disk size, etc.
 
+<WarningBox>
+
+Palette recommends Virtualized Edge deployment only when the user has a **single edge appliance and needs HA virtualized Kubernetes Cluster.**
+
+[Contact Spectro support via the Service Desk](http://support.spectrocloud.io/) for more details on the deployment of Virtualized Edge Architecture.
+
+</WarningBox>
+
 
 ### Native Edge

```

Comment:
Let's put the Native Edge section first, so that's what practitioners see first.